src/gluonts/model/estimator.py

```diff
@@ -178,6 +186,19 @@ def train_model(self, training_data: Dataset) -> TrainOutput:
             float_type=self.float_type,
         )
 
+        validation_data_loader = (
+            TrainDataLoader(
+                dataset=validation_data,
+                transform=transformation,
+                batch_size=self.trainer.batch_size,
+                num_batches_per_epoch=self.trainer.num_batches_per_epoch,
+                ctx=self.trainer.ctx,
+                float_type=self.float_type,
+            )
+            if validation_data is not None
+            else None
+        )
```

Comment:
This could be fine in case the `validation_data` consists of a different set of time series wrt training data. But even in that case, I guess one should consider batches in which the prediction time window falls entirely *after* the training dataset. This is probably tricky to obtain though...

cc @geoalgo @jgasthaus

Reply:
Yes, I've wrote it the most general way — you need to prepare correct validation yourself. 

I guess that we can implement several strategies to generate validation on the fly from the training set, but I am not sure about all of the possible options. (Cutting the most recent windows from the training data might not cover all the cases.)